experimental/generation/generator/packages/library/src/mergeAssets.ts

```diff
@@ -705,7 +703,7 @@ async function mergeDialogs(schemaName: string, oldPath: string, oldFileList: st
             insertTrigger(i, newTriggers, mergedTriggers)
             mergedTriggerMap.set(triggerName, trigger)
             if (typeof trigger === 'string') {
-                await copySingleFile(oldPath, mergedPath, triggerName + '.dialog', oldFileList, feedback)
+                await copySingleFile(newPath, mergedPath, triggerName + '.dialog', newFileList, feedback)
```

Comment:
why this changes to new path?

Reply:
Because this over new triggers which weren't in old.